data/mainnet/tokens.js

```diff
@@ -83,6 +83,26 @@ module.exports = {
     cmcRewardTokenSymbols: ['iFARM', 'DON', 'WETH'],
     disableAutoSwap: true,
   },
+  crvIbEUR: {
+    isNew: true,
+    chain: CHAINS_ID.ETH_MAINNET,
+    category: VAULT_CATEGORIES_IDS.STABLECOINS,
+    displayName: 'CRV:ibEUR',
+    apyIconUrls: ['./icons/curve-ibeur.png', './icons/cvx.png', './icons/rKP3R.png'],
+    apyTokenSymbols: ['CRV', 'CVX', 'rKP3R'],
+    logoUrl: './icons/curve-ibeur.png',
+    tokenAddress: addresses.V2.convex_ibEUR.Underlying,
+    decimals: '18',
+    vaultAddress: addresses.V2.convex_ibEUR.NewVault,
+    priceFunction: { type: GET_PRICE_TYPES.COINGECKO_CONTRACT, params: [addresses.ibEUR] },
+    estimateApyFunctions: [
+      {
+        type: ESTIMATED_APY_TYPES.CONVEX,
+        params: ['ibEUR', convexProfitSharingFactor], // how can I find out params?
```

Comment:
I don't think youve added the ibEUR pool to the `src/vaults/apys/implementations/convex.utils.js` file yet. Let me know if you need any help with that!